liblangutil/Exceptions.h

```diff
@@ -236,29 +237,26 @@ class Error: virtual public util::Exception
 		return false;
 	}
 
-	static std::string formatErrorSeverity(Severity _severity)
-	{
-		if (_severity == Severity::Info)
-			return "Info";
-		if (_severity == Severity::Warning)
-			return "Warning";
-		solAssert(isError(_severity), "");
-		return "Error";
-	}
-
-	static std::string formatErrorSeverityLowercase(Severity _severity)
+	static std::string formatErrorSeverity(Error::Severity _severity)
 	{
 		switch (_severity)
 		{
 		case Severity::Info:
-			return "info";
+			return "Info";
 		case Severity::Warning:
-			return "warning";
+			return "Warning";
 		case Severity::Error:
-			solAssert(isError(_severity), "");
-			return "error";
+			solAssert(isError(_severity));
+			return "Error";
 		}
-		solAssert(false, "");
+		solAssert(false);
```

Comment:
Gives a compiler error. `error: control reaches end of non-void function [-Werror=return-type]`

Reply:
Interesting. Looks like the compiler's right. This is still technically reachable because you could just cast some value to it, even if there's no corresponding enum value (["control reaches end of non-void function" with fully handled case switch over an enum type](https://stackoverflow.com/questions/33607284/control-reaches-end-of-non-void-function-with-fully-handled-case-switch-over-a)).

Ok then. Let's leave it as is.